src/ImageProcessorCore/Formats/Tiff/ValueDecoders/TiffIptcValueDecoder.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using ImageProcessorCore.IO;
+
+namespace ImageProcessorCore.Formats.Tiff.ValueDecoders
+{
+    internal class TiffIptcValueDecoder : ITiffValueDecoder
```

Comment:
The Value Decoders (idea ) is holding up. Kinda easy to plug-in support for IPTC in the TIFF processing pipeline.